### PR TITLE
Update production

### PR DESCRIPTION
**Pull request by kinueng**

### CODE REVIEW

File: server.xml [hunk 1]
```diff
@@ -30,7 +30,7 @@
 
     <cors
 		domain="/api/start"
-		allowedOrigins="https://www.openliberty.io,https://openliberty.io,https://draft-openlibertyio.mybluemix.net,https://staging-openlibertyio.mybluemix.net,https://ui-draft-openlibertyio.mybluemix.net,https://draft-ui-openlibertyio.mybluemix.net,https://ui-staging-openlibertyio.mybluemix.net,https://staging-ui-openlibertyio.mybluemix.net,https://demo1-openlibertyio.mybluemix.net/,https://us-south.openliberty.io/,https://us-east.openliberty.io,https://eu-gb.openliberty.io,https://eu-de.openliberty.io/,https://au-syd.openliberty.io/"
+		allowedOrigins="https://www.openliberty.io,https://openliberty.io,https://draft-openlibertyio.mybluemix.net,https://staging-openlibertyio.mybluemix.net,https://ui-draft-openlibertyio.mybluemix.net,https://draft-ui-openlibertyio.mybluemix.net,https://ui-staging-openlibertyio.mybluemix.net,https://staging-ui-openlibertyio.mybluemix.net,https://demo1-openlibertyio.mybluemix.net/,https://us-south.openliberty.io,https://us-east.openliberty.io,https://eu-gb.openliberty.io,https://eu-de.openliberty.io,https://au-syd.openliberty.io"
```
Comment (aknguyen7) [inline server.xml:33]: there is still / at the end on https://demo1-openlibertyio.mybluemix.net/

**Review by aknguyen7 — COMMENTED** [review]
the rest lgtm

File: server.xml (hunk 1 above)
Comment (kinueng) [reply server.xml:33]: Good catch

Comment (kinueng) [reply server.xml:33]: This will be resolved with #88 that will remove maintaining `starter.zip`

**Comment by kinueng** [comment]
Tested
- `unzip` warning is now gone
- Downloade SE8 + EE9.1 + MP5.0 and `./mvnw liberty:run` worked without any obvious errors

**Review by aknguyen7 — APPROVED** [review]

**Comment by kinueng** [comment]
Here is the branch to update the README.md to remove the reference to `starter.zip` https://github.com/OpenLiberty/start.openliberty.io/tree/remove-zip-instructions

**Pull request merged**